example/javalib/linting/3-palantirformat/src/A.java

```diff
@@ -1,6 +1,6 @@
 public class A {
 
-    public static void main(String[] args) {
+  public static void main(String[] args) {
     System.out.println("hello"); // indentation should be fixed
```

Comment:
TODO: Need to restore this lint error and find a way to allow it as an exception. (Can't finish today.)

Reply:
Done